docs/jam/jam/comments.rst

```diff
@@ -37,6 +39,7 @@ Multi-line Comments
 	# Having sequential single line comments is more efficient
 	# Due to modern code editing tools
 
+
```

Comment:
These spaces aren't taken literally by reST, so they don't influence the html generated. I don't think it makes the source any more readable either. Probably get rid of these.